frontend/src/api.ts

```diff
@@ -56,6 +56,9 @@ export const API = {
         DETAILS: (name: IProject['project_name']) => `${API.PROJECTS.BASE()}/${name}`,
         DETAILS_INFO: (name: IProject['project_name']) => `${API.PROJECTS.DETAILS(name)}/get`,
         SET_MEMBERS: (name: IProject['project_name']) => `${API.PROJECTS.DETAILS(name)}/set_members`,
+        ADD_MEMBERS: (name: IProject['project_name']) => `${API.PROJECTS.DETAILS(name)}/add_members`,
+        REMOVE_MEMBERS: (name: IProject['project_name']) => `${API.PROJECTS.DETAILS(name)}/remove_members`,
+        UPDATE_VISIBILITY: (name: IProject['project_name']) => `${API.PROJECTS.DETAILS(name)}/update_visibility`,
```

Comment:
I'd recommend to have `/update` instead of `/update_visibility` and reuse this API for other updates that we may want to support in the feature.